chain/bitcoind_client.go

```diff
@@ -366,10 +393,31 @@ func (c *BitcoindClient) RescanBlocks(
 	blockHashes []chainhash.Hash) ([]btcjson.RescannedBlock, error) {
 
 	rescannedBlocks := make([]btcjson.RescannedBlock, 0, len(blockHashes))
+
+	// request map will store rpcclient.FutureGetBlockHeaderVerboseResult for
+	// each blockHash.
+	request := make(
```

Comment:
nit: I think renaming this to `requests` is slightly better. Also fix formatting.